api/src/main/java/io/strimzi/api/kafka/model/authentication/KafkaClientAuthenticationOAuth.java

```diff
@@ -114,4 +116,26 @@ public boolean isDisableTlsHostnameVerification() {
     public void setDisableTlsHostnameVerification(boolean disableTlsHostnameVerification) {
         this.disableTlsHostnameVerification = disableTlsHostnameVerification;
     }
+
+    @Description("Set or limit time-to-live of the access tokens to the specified number of seconds. " +
+            "Set this if authorization server returns opaque tokens.")
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public int getMaxTokenExpirySeconds() {
+        return maxTokenExpirySeconds;
+    }
+
+    public void setMaxTokenExpirySeconds(int maxTokenExpirySeconds) {
+        this.maxTokenExpirySeconds = maxTokenExpirySeconds;
+    }
+
+    @Description("Configure if access token should not be treated as JWT. Set this to `true` if authorization " +
+            "server returns opaque tokens. Defaults to `false`.")
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public boolean isNotJwt() {
```

Comment:
I wonder if we have some better name for this. Maybe just `jwt: true` / `jwt: false`? Ot `jwtFormat`? WDYT @tombentley?

Reply:
Yeah, I wondered about this too. It's always a bit weird having a negative property and having to set notFoo=true. How about `accessTokenIsJwt` defaulting to true?

Reply:
`accessTokenIsJwt` sounds good to me 👍

Reply:
Yeah tbh I don't like "negative" properties as well :-) `accessTokenIsJwt` sound fine!

Reply:
Very well. Changed to `accessTokenIsJwt`. The same can be changed in strimzi-kafka-oauth library for its next release and the necessary modifications made here behind the scenes.